src/main/scripts/org/reaktivity/specification/nukleus/socks/streams/forward/client.connect.request.with.command.not.supported/server.rpt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()}
+property newServerConnectRef ${newClientAcceptRef} # external scope
+
+property serverAccept "nukleus://target/streams/socks#source"
+
+# Build the TcpBeginExFW using hardcoded values
+property serverLocalAddress [0x7F 0x00 0x00 0x01]
+property serverLocalPort 60123
+property serverRemoteAddress [0x7F 0x00 0x00 0x01]
+property serverRemotePort 8080
+
+accept ${serverAccept}
+  option nukleus:route  ${newServerConnectRef}
+  option nukleus:window 65536
+  option nukleus:transmission "duplex"
+accepted
+
+# Server forward mode DOES enforce target Nukleus to be TCP by sending the TcpBeginExFW
+write nukleus:begin.ext ${socks:beginTcpExtensionIpv4(serverLocalAddress, serverLocalPort, serverRemoteAddress, serverRemotePort)}
```

Comment:
Yes, please see the comment above: https://github.com/reaktivity/nukleus-socks.spec/pull/1#discussion_r149449512

Reply:
@Anisotrop David says we can use the address and port from the route (SocksRouteEx) instead. That way we can remove the dependency on tcp. Eventually this will be SocksBeginEx but we don't need to do that yet. We just need to wrap up this socks forward work so you can get onto the socks reverse stuff, which is our most pressing need (for kaazing.net).